spec/system/budgets_view_spec.rb

```diff
@@ -7,9 +7,14 @@
   let(:projects_count) { 1 }
   let(:decidim_budgets) { Decidim::EngineRouter.main_proxy(component) }
   let(:user) { create(:user, :confirmed, organization: organization) }
+  let(:sms_gateway_service) { "Decidim::Verifications::Sms::ExampleGateway" }
 
   before do
     switch_to_host(organization.host)
+
+    Decidim.configure do |config|
```

Comment:
Prefer to define mock for `Decidim.configure.sms_gateway_service` 

Example : `allow(Decidim.config.sms_gateway_service).to receive(:present?).and_return(true)`